pkg/virt-handler/vm_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"sync"
 	"time"
 
+	network_driver "kubevirt.io/kubevirt/pkg/network/driver"
```

Comment:
Remove the underscore normally at golang we just concatenate the packages and this will make it consistent with https://github.com/kubevirt/kubevirt/pull/5044/commits/162f2e94722502ae1655107118384f359c075495#diff-7a3f9e0461f554b07ac195375eb0f8db7d46579071795a65c459c721b1a8204aR66

Reply:
0_o

Reply:
About golang naming of things https://blog.golang.org/package-names